### PR TITLE
Feature/about

### PR DESCRIPTION
**Pull request by RomiKMoveo**
Added About component

made charges in padding regarding the inter homePage component.
Added icons and image for the About component.
created an About.tsx and an About.scss files.

None

I have:
- [ ] deleted all irrelevant comments, logs and debuggers.
- [ ] assure that I don't print full error stacks, full body response or headers from the server in the console.
- [ ] deleted all sensitive information from the code because it's being imported from environment variables.
- [ ] encrypted keys/passwords that are sent through the internet.
- [ ] sanitized all user inputs to prevent attacks.
- [ ] made sure my code is generic and reusable for future things or reused existing resources.
- [ ] made sure that my code is readable.
- [ ] checked that the functions and resource names are self-explanatory.
- [ ] tested the code locally and it works as expected.
- [ ] added validations and error handling in relevant cases (external requests, integrations, etc).

### CODE REVIEW

File: epicure-frontend/src/pages/home/components/About/About.tsx [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
+                lacus vel justo fermentum bibendum non eu ipsum. Cras porta
+                malesuada eros, eget blandit turpis suscipit at. Vestibulum sed
+                massa in magna sodales porta. Vivamus elit urna, dignissim a
+                vestibulum.
+              </p>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. In a
+                lacus vel justo fermentum bibendum no eu ipsum. Cras porta
+                malesuada eros.
+              </p>
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/About/About.tsx]: you do not need to split it to 2 diff p

File: epicure-frontend/src/shared/components/Button/Button.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
 .btn {
   border: 1px solid #000000;
   box-sizing: border-box;
-  padding: 5px 24px;
+  //padding: 5px 24px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/shared/components/Button/Button.scss]: remove comments

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: epicure-frontend/src/pages/home/components/About/About.tsx (hunk 1 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/About/About.tsx]: done

File: epicure-frontend/src/shared/components/Button/Button.scss (hunk 2 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/shared/components/Button/Button.scss]: done

**Pull request merged**